packages/docs/src/styles/documentation.css

```
@@ -553,20 +553,20 @@
 }
 
 /* Documentation Links - Oatmeal Colors */
-.doc-page a:not(.anchor-link) {
+.doc-page a:not(.anchor-link):not(.btn) {
```

Stylelint (17.7.0)

[error] Expected complex :not() pseudo-class notation (selector-not-notation)

(selector-not-notation)